app/code/Magento/Tax/view/frontend/web/js/view/checkout/summary/tax.js

```diff
@@ -98,6 +102,31 @@ define([
             return this.getFormattedPrice(amount);
         },
 
+        /**
+         * @param {*} parent
+         * @param {*} percentage
+         * @return {*|String}
+         */
+        getTaxAmount: function (parent, percentage) {
+            amount = parent.amount;
+            rates = parent.rates;
+            totalPercentage = 0;
+            _.each(rates, function (rate) {
+                totalPercentage += parseFloat(rate.percent); 
+            });
+            return this.getFormattedPrice(this.getPercerntAmount(amount, totalPercentage, percentage));
+        },
+
+        /**
+         * @param {*} amount
+         * @param {*} totalper
+         * @param {*} per
+         * @return {*|String}
+         */
+        getPercerntAmount: function (amount, totalper, per) {
```

Comment:
small typo here "percernt"
also, try to avoid abbreviations, variables could also be named totalPercentage and percentage